frontend/settings/OBSBasicSettings.cpp

```diff
@@ -632,7 +632,7 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	ui->enableNewSocketLoop = nullptr;
 	ui->enableLowLatencyMode = nullptr;
 	ui->hideOBSFromCapture = nullptr;
-#ifdef __linux__
+#if !defined(__APPLE__) && !defined(__linux__)
```

Comment:
Bit confused by this one - previously linux only, becomes Windows only?

Reply:
Line 590 `#ifdef _WIN32` and line 610 `#else`, it becomes FreeBSD/OpenBSD only.

This PR has a byproduct, it fixes the presence of the toggle on FreeBSD/OpenBSD.

Reply:
Ah, makes sense.